scene/gui/tree.cpp

```diff
@@ -1222,6 +1222,12 @@ int TreeItem::get_button_by_id(int p_column, int p_id) const {
 	return -1;
 }
 
+Color TreeItem::get_button_color(int p_column, int p_index) const {
+	ERR_FAIL_INDEX_V(p_column, cells.size(), Color::named("TRANSPARENT"));
+	ERR_FAIL_INDEX_V(p_index, cells[p_column].buttons.size(), Color::named("TRANSPARENT"));
```

Comment:
No reason to do this. The documentation can state the constant, but internally it can still be an empty Color constructor (as it should be).

Reply:
Except the constructor `Color()` returns the equivalent of `Color.BLACK` (or `Color(0, 0, 0, 0)` in C#)
so do I return a transparent color or update the docs to reference `Color()`?

Reply:
No it returns `Color(0, 0, 0, 1)`

Reply:
Alright, it's settled then. `Color()` internally, and the documentation should refer to `Color.BLACK`. Forgive me, the original statement of the PR confused me.